.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,15 @@
+name: reviewdog
```

Comment:
Would be preferable to integrate this into the existing `tests.yml` and just run `brew install shellcheck` there, thanks. You may even be able to just use `brew style`.

Reply:
Fixed by b38ead0ba0d665e4d7d833a21b7ca448c08a8806